examples/01_tutorials/plot_single_subject_single_run.py

```diff
@@ -90,13 +90,13 @@
 # * standardize=False means that we do not want to rescale the time series to mean 0, variance 1
 # * hrf_model='spm' means that we rely on the SPM "canonical hrf" model (without time or dispersion derivatives)
 # * drift_model='cosine' means that we model the signal drifts as slow oscillating time functions
-# * period_cut=160(s) defines the cutoff frequency (its inverse actually).
+# * high_pass=1/160(Hz) defines the cutoff frequency.
 fmri_glm = FirstLevelModel(t_r=7,
                            noise_model='ar1',
                            standardize=False,
                            hrf_model='spm',
                            drift_model='cosine',
-                           period_cut=160)
+                           high_pass=1./160)
 
 ###############################################################################
 # Now that we have specified the model, we can run it on the fMRI image
```

Comment:
All the files below have been mistakenly introduced (?)

Reply:
Yeah. I have commented about that.

Reply:
I am really sorry. I hope to have fixed it.

Reply:
Tests in CircleCI fail if I change here high_pass to``` .01``` (or ```1./128```).

```
Unexpected failing examples:
/home/analu/mygit/nistats/examples/01_tutorials/plot_single_subject_single_run.py failed leaving traceback:
Traceback (most recent call last):
  File "/home/analu/mygit/nistats/doc/sphinxext/sphinx_gallery/gen_rst.py", line 450, in execute_code_block
    exec(code_block, example_globals)
  File "<string>", line 3, in <module>
  File "/home/analu/mygit/nistats/nistats/first_level_model.py", line 569, in compute_contrast
    con_vals, stat_type)
  File "/home/analu/mygit/nistats/nistats/contrasts.py", line 96, in _fixed_effect_contrast
    contrast_ = compute_contrast(lab, res, con_val, contrast_type)
  File "/home/analu/mygit/nistats/nistats/contrasts.py", line 63, in compute_contrast
    resl = regression_result[label_].Tcontrast(con_val)
  File "/home/analu/mygit/nistats/nistats/model.py", line 189, in Tcontrast
    matrix.shape[1]))
ValueError: t contrasts should be length P=16, but this is length 11
```

Any ideas why?

Reply:
> I am really sorry. I hope to have fixed it.

It is a common mistake, easily fixed.

Well 0.01 is not 1./160 and the examples are tailored to a period_cut of 160, so it shouldn't be a surprise that they broke. We need to stick with more acuuracy or precision or modify the examples accordingly, however that may be.

Reply:
I think that the example should allow a broad range of cutoff frequencies, encompassing the standard value. @bthirion

Reply:
Btw, the code starts failing from period_cuts below 150s.

Reply:
Perhaps, we should keep the original value, i.e. 1/160, and open an issue about this. I don't think this PR is related to this problem.

Reply:
As much as we increase the high_pass frequency, the length P of the Tcontrast vector increases. This example only accepts P =< 11, otherwise it raises an error. It thus raises an error for high_pass frequencies > 1/150, because P >=12. Any suggestion how to solve this? @bthirion @kchawla-pi (please, see log error in my comment above)

Reply:
Simply pad the contrast vectors (l. 156-157) with 5 more zeros (as suggested by the error message).
Best,

Reply:
OK. I fixed it. I guess it shall be ready to be merged. Nevertheless, if the high cutoff frequency is changed by the user, the length of the contrast vectors shall also be changed accordingly. I guess this should be done automatically. We can open an issue concerning this.

Reply:
The problem discussed in this thread of this PR has been addressed in the issue #354 and it will be solved in a separate PR. @kchawla-pi and @bthirion, are there any further developments needed in this PR?